<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiplayerWindow</class>
 <widget class="QWidget" name="MultiplayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>278</width>
        <height>198</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="info">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="listBtn">
           <property name="text">
            <string>挑战列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="callGameBtn">
           <property name="text">
            <string>发起挑战</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backMenuBtn">
           <property name="text">
            <string>返回菜单</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>服务器地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ip"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>服务器端口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="port"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="connectBtn">
           <property name="text">
            <string>连接服务器</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
